CHANGELOG.md

```diff
@@ -5,12 +5,21 @@
 accidentally triggering the load of a previous DB version.**
 
 ## To be released
+Users who upgrade from 1.7 and have continuous aggregateswith real time support set up should follow these instructions
+
+Step 1: Find views that have real time support enabled
+select view_name, materialized_only from timescaledb_information.continuous_aggregates where materialized_only = 'true';
```

Comment:
this query is wrong, only views with materialized_only = false are affected

Reply:
modified changelog.